### DISCUSSION TITLE
/spawn Command & Worldspawn under Blocks

### DISCUSSION THREAD

**Discussion by Nick-Tricks** [PurpurMC/Purpur#1510] [Feature Requests]
Hello, I have an idea for a command that teleports a player to the worldspawn. maybe even with a coldown to teleport.

I also have an idea for an option so that the worldspawn can also be under blocks.

**Comment by Onako2** [comment]
The world spawn is on top because the player shouldn't instantly die on spawn. It would be interesting what would happen but it is also risky if it should be a SMP and the owner doesn't know what the option does

**Reply by Nick-Tricks** [reply]
You could write that you should only activate it if you really know what it is

**Comment by Krakenied** [comment]
Definitely it's more a plugin thing. Adding spawn command directly to the server would probably cause incompatibilities with plugins already having that command

**Reply by Nick-Tricks** [reply]
OK, that makes sense
